sklearn/preprocessing/_encoders.py

```diff
@@ -267,12 +271,22 @@ class OneHotEncoder(_BaseEncoder):
     >>> enc.get_feature_names(['gender', 'group'])
     array(['gender_Female', 'gender_Male', 'group_1', 'group_2', 'group_3'],
       dtype=object)
+
+    One can always drop the first column for each feature:
+
     >>> drop_enc = OneHotEncoder(drop='first').fit(X)
     >>> drop_enc.categories_
     [array(['Female', 'Male'], dtype=object), array([1, 2, 3], dtype=object)]
     >>> drop_enc.transform([['Female', 1], ['Male', 2]]).toarray()
     array([[0., 0., 0.],
            [1., 1., 0.]])
+
+    Or only drop a column for feature having only 2 categories:
```

Comment:
Also too many "only" IMO )

Reply:
At least this is quite enforced :), I'll make the change